assemblies/assembly-planner-configuration.adoc

```diff
@@ -596,12 +598,8 @@ To configure your Spring Boot OptaPlanner  application to use Micrometer and a s
  <artifactId>micrometer-registry-<MONITORING_SYSTEM></artifactId>
 </dependency>
 ----
-. Add the following line to the application's `application.properties` file:
-+
-[source]
-----
-management.endpoints.web.exposure.include=metrics,<MONITORING_SYSTEM>
-----
+. Add configuration information to the application's `application.properties` file. For information, see the https://micrometer.io[Micrometer] web site.
```

Comment:
For Spring Boot, the website is https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.metrics.export